superset-frontend/src/filters/components/Range/RangeFilterPlugin.tsx

```diff
@@ -33,12 +33,13 @@ export default function RangeFilterPlugin(props: PluginFilterRangeProps) {
   const [row] = data;
   // @ts-ignore
   const { min, max }: { min: number; max: number } = row;
-  const { groupby } = formData;
-  const [col] = groupby || [];
+  const { groupby, currentValue, defaultValue } = formData;
+  const [col = ''] = groupby || [];
+  const [value, setValue] = useState<[number, number]>(defaultValue ?? [0, 0]);
```

Comment:
I think this should default to `min` and `max` respectively
```suggestion
  const [value, setValue] = useState<[number, number]>(defaultValue ?? [min, max]);
```